docs/howto/operate/node-administration.md

```diff
@@ -0,0 +1,44 @@
+# Manual node administration
+
+The current pilot Kubernetes clusters we run, have two types of group nodes ([node pools](https://cloud.google.com/kubernetes-engine/docs/concepts/node-pools)), configured to run on: `core` and `user` pools.
+
+The [node selector](https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#nodeselector) configuration will ensure the following assignment of pods to nodes:
+
+- **Core and support pods** will be assigned nodes from the core pool
+- **User pods** will be assigned to nodes in the user pool.
+
+This separation should protect against user pods exhausting the resources needed by core and support pods.
+
+The machines where the core nodes run, are different than the ones on which the user nodes run. The type of these machines is chosen based on the number, type and the resource needs (CPU, memory, etc.) of the pods that will be scheduled to run on these nodes. Because of this resource dependance, these types might be adjusted in the future. You can checkout the exact type of the core and user nodes VMs in the [`terraform` config](https://github.com/2i2c-org/pilot-hubs/blob/master/terraform/variables.tf).
```

Comment:
```suggestion
The machines where the core nodes run, are different than the ones on which the user nodes run.
The type of these machines is chosen based on the number, type, and the resource needs (CPU, memory, etc.) of the pods that will be scheduled to run on these nodes.
Because of this resource dependance, these types might be adjusted in the future.
You can checkout the exact type of the core and user nodes VMs in the [`terraform` config](https://github.com/2i2c-org/pilot-hubs/blob/master/terraform/variables.tf).
```

Reply:
I also split up this paragraph so that it is "one sentence per line". I've found this is a nice compromise between "one paragraph per line" and a strict "77 characters per line" for documentation.